src/components/MyProfile/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import {
+  SideNavLink
+} from '@carbon/react';
+import * as Icons from '@carbon/icons-react';
+
+import AvatarImage from '../AvatarImage';
+import PanelSectionName from '../PanelSectionName';
+
+import UserData from '../../mock-data/UserData';
+import AccountOptions from '../../mock-data/AccountOptions';
+
+import './style.scss';
+
+const MyProfile = () => {
+  const navigate = useNavigate();
+
+  const goTo = React.useCallback((url: string) => {
+    navigate(url);
+  }, []);
+
+  const goOut = React.useCallback(() => {
+    navigate('/logout');
+  }, []);
+
+  return (
+    <>
+      <div className="user-info-section">
+        <div className="user-image">
+          <AvatarImage userName={UserData.name} size="large" />
+        </div>
+        <div className="user-data">
+          <p className="user-name">{UserData.name}</p>
+          <p>{`IDIR: ${UserData.idir}`}</p>
+          <p>{UserData.email}</p>
```

Comment:
I think we can replace these mocked data with real data from the logged user.

Reply:
Yes, we can. I'll adjust that 🐱‍🏍.